### PR TITLE
Update Feast to support 0.18.x and later

### PR DESCRIPTION
**Pull request by cbtham**
- Add support for feast 0.18.x and later
	- Updated azure_provider to support new methods in 0.18.x and later
	- Updated mssqlserver to support new methods in 0.18.x and later
- Artificially bump up version of feast-azure-provider
- Added a note section for contributors to build, install and run feast-azure-provider

Note: Latest feast version tested working is 0.18.1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -357,3 +357,8 @@ config.json
```
Comment (samuel100) [inline .gitignore:357]: What is this file doing exactly?

File: provider/README.md [hunk 2]
```diff
@@ -35,6 +35,18 @@ The only 2 required parameters during the set-up are:
...
+```bash
+python setup.py bdist_egg
+```
+to generate an installable .egg file.
+
+If you would like to install or run a custom version of feast-azure-provider, go to provider/sdk where setup.py is located, run  
+```bash
+python setup.py install
+```
+
```
Comment (samuel100) [inline provider/README.md:50]: Can you elaborate on what a custom version of the feast-azure-provider is?

Comment (cbtham) [reply provider/README.md:50]: This is to let contributors build and install their own feast-azure-provider for testing. At the moment the docs does not mention how to build and install feast-azure-provider outside of pypi repository - https://pypi.org/project/feast-azure-provider/

File: .gitignore (hunk 1 above)
Comment (cbtham) [reply .gitignore:357]: gitignore file and folder generated by recompiling feast-azure-provider

**Pull request merged**